test/acceptance/features/unbindAppToService.feature

```diff
@@ -0,0 +1,55 @@
+Feature: Unbind an application from a service
+
+    As a user of Service Binding Operator
+    I want to unbind an application from a service
+
+    Background:
+        Given Namespace [TEST_NAMESPACE] is used
+        * Service Binding Operator is running
+
+    @disabled
+    Scenario: Unbind a generic test application from database
```

Comment:
we do not use database anymore, please update the description.

Reply:
Done